cfgov/paying_for_college/disclosures/scripts/update_ipeds.py

```diff
@@ -189,16 +186,16 @@ def create_school(id, data):
     school = School(school_id=id, data_json=STARTER_DATA_JSON)
     for field in data:
         if field == 'alias':
-            ALIAS = data['alias']
+            _alias = data['alias']
         else:
             setattr(school, field, data[field])
     school.zip5 = school.zip5[:5]
     school.save()
-    create_alias(ALIAS, school)
+    create_alias(_alias, school)
```

Comment:
Won't this code fail if the first branch of the `if` never gets hit, and `_alias` (or previously `ALIAS`) is undefined?

Reply:
Excellent point. It's never happened, but that logic is bad.  
Fixed.